### PR TITLE
Added Disable Schedule feature + InputField Fixes + Modifications

### PR DESCRIPTION
**Pull request by S-ayanide**
## Proposed changes

* Added Disable Schedule Feature
* Fixes minor Breadcrumb + Route issue
* Added onClick InfoToggler and Node renamed to Step
* Dropdown and InputField fixes
* Workflow Name Validation updated
* Fixes WeightSlider by @amityt 

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [X] New feature (non-breaking change which adds functionality)
- [X] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [X] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [X] I have signed the commit for DCO to be passed.
- [X] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx [hunk 1]
```diff
@@ -111,46 +111,88 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
...
-                : cronstrue.toString(data.cronSyntax)}
+              <span
+                className={
+                  YAML.parse(data.workflow_manifest).spec.suspend === true
+                    ? classes.dark
+                    : ''
+                }
+              >
+                {data.cronSyntax === ''
+                  ? 'Once'
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx:151]: Translation required @S-ayanide

File: litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx [hunk 2]
```diff
@@ -111,46 +111,88 @@ const TableData: React.FC<TableDataProps> = ({ data, deleteRow }) => {
...
+                </Typography>
+                <KeyboardArrowDownIcon className={classes.expInfoActiveIcon} />
+              </div>
+            ) : (
+              <div className={classes.expDiv}>
+                <Typography className={classes.expInfo}>
+                  <strong>Show Experiment</strong>
+                </Typography>
+                <ChevronRightIcon />
+              </div>
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/ChaosWorkflows/BrowseSchedule/TableData.tsx:193]: Translations required @S-ayanide

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx [hunk 3]
```diff
@@ -97,7 +98,7 @@ const VerifyCommit: React.FC<VerifyCommitProps> = ({
       annotations: editorValidations.annotations,
     };
     if (stateObject.annotations.length > 0) {
-      setYamlStatus('Error in CRHeloD Yaml.');
+      setYamlStatus('Error in CRD Yaml.');
     } else {
       setYamlStatus('Your code is fine. You can move on !');
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx:103]: use from translation

File: litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx [hunk 4]
```diff
@@ -192,7 +193,11 @@ const VerifyCommit: React.FC<VerifyCommitProps> = ({
                 ampm
                 disabled={edit}
               /> */}
-              {cronSyntax === '' ? (
+              {isDisabled ? (
+                <Typography className={classes.schedule}>
+                  Currently Disabled
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/VerifyCommit/index.tsx:198]: use translation

**Review by ishangupta-ds — CHANGES_REQUESTED** [review]
@S-ayanide PTAL, translations are required. Rest LGTM.

**Review by amityt — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by Jonsy13 — APPROVED** [review]

**Pull request merged**